source/modules/MEI.shared.xml

```diff
@@ -757,7 +757,13 @@
             <desc>Downward curve.</desc>
           </valItem>
           <valItem ident="mixed">
-            <desc>A "meandering" curve, both above and below the items it pertains to.</desc>
+            <desc>A "meandering" curve, both above and below the items it pertains to, starting un-determinately above or below.</desc>
```

Comment:
I am wondering if "un-determinately" could give the false impression that it could not be determined if a curve starts above or below. Would "starting either above or below without specific indication." work?

Maybe a native speaker can clarify?

Reply:
It works for me